### PR TITLE
Add Manage_Watched_Domains_with_Darktrace_from_Slack workflow

### PR DESCRIPTION
**Pull request by ncramer-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

workflow

workflow_loop

#### Job

slack_add_success
slack_remove_success
slack_failure

#### Artifact

artifact

## Validation
```
ncramer@BOS-MBP-8176:~/go/src/github.com/rapid7/insightconnect-workflows/workflows/Manage_Hosts_with_Darktrace_from_Slack|darktrace-slack-workflow ⇒  icon-validate .
[*] Validating workflow at .

[*] Running Integration Validators...
[*] Executing validator WorkflowDirectoryNameMatchValidator
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*] Workflow successfully validated!

----
[*] Total time elapsed: 19.282ms
```

### CODE REVIEW

**Comment by ncramer-r7** [comment]
Darktrace may need a new plugin icon -- currently it is hard to see in the product since it is black.

**Review by jschipp-r7 — CHANGES_REQUESTED** [review]
Waiting to hear back from Darktrace on whether Domains are only supported. If so, we will need to update all the naming and language.

File: workflows/Manage_Hosts_with_Darktrace_from_Slack/workflow.spec.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+extension: workflow
+products: ["insightconnect"]
+name: Manage_Hosts_with_Darktrace_from_Slack
+title: Manage Hosts with Darktrace from Slack
```
Comment (jschipp-r7) [inline workflows/Manage_Hosts_with_Darktrace_from_Slack/workflow.spec.yaml:4]:
```suggestion
title: Manage Watched Domains with Darktrace from Slack
```

File: workflows/Manage_Hosts_with_Darktrace_from_Slack/workflow.spec.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+extension: workflow
+products: ["insightconnect"]
+name: Manage_Hosts_with_Darktrace_from_Slack
```
Comment (jschipp-r7) [inline workflows/Manage_Hosts_with_Darktrace_from_Slack/workflow.spec.yaml:3]:
```suggestion
name: Manage_Watched_Domains_with_Darktrace_from_Slack
```

File: workflows/Manage_Hosts_with_Darktrace_from_Slack/workflow.spec.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+title: Manage Hosts with Darktrace from Slack
+description: Add or remove hosts in the Darktrace watched domains list from Slack.
```
Comment (jschipp-r7) [inline workflows/Manage_Hosts_with_Darktrace_from_Slack/workflow.spec.yaml:5]:
```suggestion
description: Add or remove domains and IP addresses in the Darktrace watched domains list from Slack.
```

File: workflows/Manage_Hosts_with_Darktrace_from_Slack/workflow.spec.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [threat_detection_and_response]
+  keywords: [chat, chatops, edr, endpoint, darktrace, work_from_home, slack]
```
Comment (jschipp-r7) [inline workflows/Manage_Hosts_with_Darktrace_from_Slack/workflow.spec.yaml:12]:
```suggestion
  keywords: [chat, chatops, darktrace, nta, nsm, work_from_home, slack]
```

**Review by jschipp-r7 — CHANGES_REQUESTED** [review]

**Review by jschipp-r7 — APPROVED** [review]

**Pull request merged**